adapters/solana/solana_da_programs/Anchor.toml

```diff
@@ -0,0 +1,17 @@
+[features]
```

Comment:
usually our file names start with a small letter.

Reply:
ah, that's auto generated by anchor - it will get upset if i change it. Its analogous to Cargo.toml I would imagine